### PR TITLE
fix : Travis build works now

### LINKED ISSUE #573: Travis CI does not run anymore
The Travis CI build does not run anymore for recent PRs, such as e.g. #567.

The README points to the old wrong TravisCI.org instead of TravisCI.com which is now used.

https://github.com/openMF/mobile-wallet/commits/dev has links to CloudBees Codeship, e.g. https://app.codeship.com/projects/332704/builds/b17ee1e9-ffe3-4c9f-98ea-1dc2cafcf059, but those do not appear to be public?  Also the README only links to and mentions TravisCI, not Codeship.

I'll see if I can do anything quick & easy to make TravisCI kick in again...

@edcable @Bhavnaharitsa @jsahil730 @luckyman20 FYI, feel free to chime in here, if anything

### PR DESCRIPTION
**Pull request by jsahil730**
Updated pmd-rulesets and sorted indentation-issues.

Fixes #573 and #566 

Please Add Screenshots If any UI changes.
None
Please Add Summary of what changes you have made.(Optional)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: config/quality/pmd/pmd-ruleset.xml [hunk 1]
```diff
@@ -1,35 +1,37 @@
 <?xml version="1.0"?>
 <ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Android Application Rules"
-         xmlns="http://pmd.sf.net/ruleset/1.0.0"
-         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd"
-         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd">
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0
+          https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
 
     <description>Custom ruleset for Mifos Self-Service Android application</description>
```
Comment (luckyman20) [inline config/quality/pmd/pmd-ruleset.xml]:
```suggestion
    <description>Custom ruleset for Mifos Pay Android application</description>
```

File: mifospay/build.gradle [hunk 2]
```diff
@@ -75,6 +75,8 @@ dependencies {
     implementation "io.reactivex:rxandroid:$rootProject.rxandroidVersion"
     implementation "io.reactivex:rxjava:$rootProject.rxjavaVersion"
 
+    implementation "io.michaelrocks:libphonenumber-android:8.11.0"
```
Comment (luckyman20) [inline mifospay/build.gradle]: We would have to close #567 if you add this but I think that would be wiser option. We would still cosider @Bhavnaharitsa 's contribution. So mention in the PR desc that it fixes that issue too.

**Comment by jsahil730** [comment]
@luckyman20 pmd rulesets have been updated afaik, so we could move on to the newer rules if that's fine.

**Comment by luckyman20** [comment]
pmd

I guess `deprecated` means that it is no longer updated although we can use it but it would be more beneficial if we move to something which is regularly maintained. I am not trying to support Codeship in any ways but I feel if it is free for open source then we should definitely use it (or something like that).

**Comment by luckyman20** [comment]
@jsahil730 Please squash the commits.

**Pull request merged**